docs/testplan/dual_tor/dual_tor_orch_test_plan.md

```diff
@@ -0,0 +1,115 @@
+# Dual ToR Orchangent Test Plan
+
+### Scope
+
+The scope of this test plan is to verify correct tunnel, ecmp, acl behavior. This is a continuation to dual_tor_test_hld.md and please refer this documention for more details on topology etc. 
+
+Standby ToR refers to a ToR scenario where the packet is destined to/from server where the Mux is in Standby Mode
+Active ToR refers to a ToR scenario where the packet is destined to/from server where the Mux is in Active Mode
+
+### Testbed Setup
+
+The test can be executed on a single tor testbed with suggest configuration that simulates the active/standby mux behavior
```

Comment:
Could we add a non-test case item that will automatically apply this config work for us? I can also add this to the main test plan doc instead since there are already several non-test case items.

Reply:
Sure, can be added in main doc